chart/flux/templates/deployment.yaml

```diff
@@ -99,6 +99,18 @@ spec:
           - name: http
             containerPort: 3030
             protocol: TCP
+          livenessProbe:
+            httpGet:
+              port: 3030
+              path: /api/flux/v6/identity.pub
```

Comment:
Given our quest earlier today, this is a smart intermediate solution :1st_place_medal: